docs/NAMESPACE_RESTRICTED_METRICS.md

```
@@ -115,13 +115,20 @@ curl "https://.../api/v1/query?query=up&namespace=my-namespace"
 Forbidden (user=<username>, verb=get, resource=pods, subresource=)
 ```
 
-**Cause**: User lacks permissions for `metrics.k8s.io/pods` resource in the requested namespace
+or (for POST requests, e.g. from Perses):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)